src/components/commons/AbstractSelect/AbstractSelect.js

```
@@ -7,7 +7,7 @@ const { Option } = Select;
 
 function AbstractSelect(props) {
   return (
-    <Select className={props.className} onChange={v => props.onChange(v)}>
+    <Select className={props.className} onChange={console.log(1)}>
```

Pls remove the `console.log` in onChange prop. The `AbstractSelect` should execute the `onChange` function once its value changed. And just by the way, the `AbstractSelect` should throw an error once its value changes as `console.log`(which returns `null`) is not a function.